core/src/main/resources/photonOperatorMapping.json

```diff
@@ -0,0 +1,99 @@
+{
+  "PhotonAdapter": [
+    "Scan"
+  ],
+  "PhotonScan": [
+    "Scan"
+  ],
+  "PhotonSubqueryBroadcast": [
+    "SubqueryBroadcast"
+  ],
+  "PhotonResultStage": [
+    "WholeStageCodegen"
+  ],
+  "PhotonUnionShuffleExchangeSink": [
+    "Union"
+  ],
+  "PhotonUnionShuffleMapStage": [
```

Comment:
this seems like an odd mapping to go to whole stage code gen. Wholestagecode gen isn't an exec so is the intention here to say this is more metadata?
This goes for all of the ones mapped to WholeStageCodeGen

Reply:
Yes Tom. All `Photon*MapStage` are basically a metadata which is similar to `WholeStageCodeGen`.

Attached an image for Reference:

![Databricks Shell - Details for Query 28](https://github.com/user-attachments/assets/4c1403c7-2d64-438d-a7c4-0a1c5cfc9c6d)